spyder_kernels/ipdb/tests/test_ipdb_kernel.py

```diff
@@ -231,6 +231,13 @@ def test_ls_path_complete(ipdb_kernel):
         assert comp['matches'] == ['"ipython\\"'], comp
 
 
+def test_no_path(ipdb_kernel):
+    kernel = ipdb_kernel
+    info = {'code': '~/.ipytho'}
```

Comment:
@dalthviz, let's better get completions for `/tm` and assert that there's only one completion returned.

Also, skip this test for Windows (given that `/tmp` only exists on Linux).